engine/src/main/java/org/terasology/world/chunks/localChunkProvider/LocalChunkProvider.java

```diff
@@ -163,6 +167,15 @@ public ChunkViewCore getLocalView(Vector3i centerChunkPos) {
         return null;
     }
 
+    @Override
+    public ChunkViewCore getLocalView(Vector3ic centerChunkPos) {
+        BlockRegion region = new BlockRegion().setMin(centerChunkPos).addExtents(ChunkConstants.LOCAL_REGION_EXTENTS.x, ChunkConstants.LOCAL_REGION_EXTENTS.y, ChunkConstants.LOCAL_REGION_EXTENTS.z);
```

Comment:
Shouldn't this just be an AABBi, as it isn't actually the block coordinates this refers to? It seems a bit semantically incorrect.